tests/unit_tests/test_data_types.py

```diff
@@ -1430,6 +1431,63 @@ def test_partitioned_table():
     assert partition_table != wandb.data_types.PartitionedTable(parts_path="parts2")
 
 
+################################################################################
+# Test wandb.Html
+################################################################################
+
+
+def test_wandb_html_with_directory():
+    dir = tempfile.gettempdir()
```

Comment:
Use the pytest [tmp_path](https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture) fixture instead throughout the test.